CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+Want to contribute? Any release of Arizona Bootstrap is tested and ready to use, but improvements to the framework are in everyone's best interest.
+
+If you want to get involved, you can:
+
+<ul>
+  <li>Request access to join the discussion on Slack (see below)</li>
+  <li>Participate in Friday meetings and Wednesday pull request review sessions (ask for more details in Slack)</li>
+  <li>Submit pull requests on GitHub</li>
+</ul>
+
+Questions, bug reports or suggestions can also be emailed to [uadigital@email.arizona.edu](mailto:uadigital@email.arizona.edu) or (preferably!) reported on Slack.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Installation 
+
+Include Arizona Bootstrap’s source Sass and JavaScript files via npm, Composer or Meteor. Package managed installs don’t include documentation, but do include our build system and readme.
+
+`npm install https://github.com/az-digital/arizona-bootstrap`
+
+## Pull Request Process
+
+1. Create an issue on GitHub with the appropriate [issue type template](https://github.com/az-digital/arizona-bootstrap/issues/new/choose). 
+2. Create a branch on the arizona-bootstrap repository using the issue type naming convention of `<issue-type>/<issue-number>`. Example: `feature/123` or `bug/123`.
+3. Make and commit your changes.
+4. Create a pull request to be reviewed by the Arizona Digital community. Be sure to include a reference to the issue in your pull request description. Read more about [linking a pull request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
+
+## Policies
+
+Contributions to Arizona Bootstrap must adhere to Arizona Digital's policies and Code of Conduct.
+
+- [Arizona Digital merging policies](https://github.com/az-digital/policies/blob/master/code-of-conduct.md)
```

Comment:
I don't remember what we were trying to say with the Arizona Digital merging policies link.

Reply:
I think we just decided to link to the merging policies since it has to be reviewed and approved?